docs-new/docs/graphql/core/api-reference/schema-api/index.mdx

```diff
@@ -0,0 +1,93 @@
+
+## Request structure
+
+```http
+POST /v2/query HTTP/1.1
+
+{
+   "type": "<query-type>",
+   "args": <args-object>
+}
+```
+
+### Request body
+
+[Query](#schema-query)
+
+#### Query{#schema-query}
+
+| Key     | Required | Schema     | Description                     |
+|---------|----------|------------|---------------------------------|
+| type    | true     | String     | Type of the query               |
+| args    | true     | JSON Value | The arguments to the query      |
+| version | false    | Integer    | Version of the API (default: 1) |
+
+## Request types
+
+The various types of queries are listed in the following table:
+
+| `type`                                                                               | `args`                                                                                       | `version` | Synopsis                                      |
+|--------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-----------|-----------------------------------------------|
+| **bulk**                                                                             | [Query](#query) array                                                                        | 1         | Execute multiple operations in a single query |
+| [run_sql](/graphql/core/api-reference/schema-api/run-sql.mdx#schema-run-sql}) | [run_sql_args](/graphql/core/api-reference/schema-api/run-sql.mdx#schema-run-sql-syntax)     | 1         | Run SQL directly on Postgres                  |
+
+## Response structure
+
+| Status code	 |  Description	           |  Response structure                        |
+|--------------|-------------------------|--------------------------------------------|
+| 200	         |	Success                |  Request specific                          |
+| 400	         |	Bad request            |  `{ "path"  : String, "error" : String}`   |
+| 401	         |	Unauthorized           |  `{"error" : String}`                      |
+| 500	         |	Internal server error	 |  `{"error" : String}`                      |
+
+## Disabling schema API
+
+Since this API can be used to make changes to the GraphQL schema, it can
+be disabled, especially in production deployments.
+
+The `enabled-apis` flag or the `HASURA_GRAPHQL_ENABLED_APIS` env var can
+be used to enable/disable this API. By default, the schema/metadata API
+is enabled. To disable it, you need to explicitly state that this API is
+not enabled i.e. remove it from the list of enabled APIs.
+
+```bash
+# enable only graphql api, disable metadata and pgdump
+--enabled-apis="graphql"
+HASURA_GRAPHQL_ENABLED_APIS="graphql"
+```
+
+See [GraphQL Engine server config reference](/graphql/core/deployment/graphql-engine-flags/reference.mdx#server-flag-reference) for info on setting the above flag/env var.
+
+- [Run SQL](<run-sql>)
```

Comment:
this is hidden in original rst. So, lets remove

Reply:
Done